src/test/kotlin/uk/gov/dluhc/notificationsapi/testsupport/testdata/dto/NotificationDtoBuilder.kt

```diff
@@ -0,0 +1,47 @@
+package uk.gov.dluhc.notificationsapi.testsupport.testdata.dto
+
+import uk.gov.dluhc.notificationsapi.dto.CommunicationChannel
+import uk.gov.dluhc.notificationsapi.dto.NotificationDto
+import uk.gov.dluhc.notificationsapi.dto.NotificationType
+import uk.gov.dluhc.notificationsapi.dto.NotifyDetailsDto
+import uk.gov.dluhc.notificationsapi.dto.PostalAddress
+import uk.gov.dluhc.notificationsapi.dto.SourceType
+import uk.gov.dluhc.notificationsapi.testsupport.testdata.aGssCode
+import uk.gov.dluhc.notificationsapi.testsupport.testdata.aLocalDateTime
+import uk.gov.dluhc.notificationsapi.testsupport.testdata.aNotificationId
+import uk.gov.dluhc.notificationsapi.testsupport.testdata.aRequestor
+import uk.gov.dluhc.notificationsapi.testsupport.testdata.aSourceReference
+import uk.gov.dluhc.notificationsapi.testsupport.testdata.anEmailAddress
+import java.time.LocalDateTime
+import java.util.UUID
+
+fun aNotificationDtoBuilder(
```

Comment:
I think these are usually named like `buildNotificationDto`

Same comment for NotifyDetailsDtoBuilder

Reply:
Not in NotificationSummaryBuilder which is the one I checked 🙂 
You're right that they're named "build..." elsewhere though, I'll change the ones that I've added as well as the NotificationSummaryBuilder.